easybuild/easyconfigs/b/BEDTools/BEDTools-2.27.1-foss-2016b.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.27.1'
+
+homepage = "https://github.com/arq5x/%(namelower)s2"
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/arq5x/%(namelower)s2/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TGZ]
```

Comment:
@SethosII This should be `SOURCELOWE_TAR_GZ` instead?

Also, please include a SHA256 checksum.

Reply:
@boegel I changed this.